src/audiences/preview/components/Selector.js

```diff
@@ -52,6 +52,11 @@ export default function Selector() {
 			setSelected( nextSelected );
 			Altis.Analytics.overrideAudiences( nextSelected );
 		}
+
+		// Ensure images are loaded with GaussHolder if available.
+		if ( window.GaussHolder ) {
+			window.GaussHolder();
+		}
```

Comment:
This code should be in the XBs themselves, triggered whenever the content is set / changed. You may still find race conditions without that specific coupling of the event order.